symfony/plugins/orangehrmCorePlugin/config/CorePluginConfiguration.php

```diff
@@ -102,5 +104,7 @@ private function registerCoreSubscribers(): void
         $dispatcher->addSubscriber(new RequestBodySubscriber());
         $dispatcher->addSubscriber(new ModuleUnderDevelopmentSubscriber());
         $dispatcher->addSubscriber(new MailerSubscriber());
+        $dispatcher->addSubscriber(new ModuleNotAvailableSubscriber());
+        $dispatcher->addSubscriber(new APINotAvailableSubscriber());
```

Comment:
Can't we merge both together, Seems duplicating

Reply:
done